src/views/login/login.view.tsx

```diff
@@ -0,0 +1,59 @@
+import Card from "src/views/shared/card/card.view";
+import useLoginStyles from "src/views/login/login.styles";
+import { ReactComponent as MetaMaskIcon } from "src/assets/icons/metamask.svg";
+import { ReactComponent as WalletConnectIcon } from "src/assets/icons/walletconnect.svg";
+import Typography from "src/views/shared/typography/typography.view";
+import { ReactComponent as CaretRightIcon } from "src/assets/icons/caret-right.svg";
+import { ReactComponent as PolygonHermezLogo } from "src/assets/polygon-hermez-logo.svg";
+
+const Login = () => {
+  const classes = useLoginStyles();
+  const walletsData = [
+    {
+      name: "MetaMask",
+      description: "Connect using web wallet",
+      Icon: MetaMaskIcon,
+      iconContainerClassName: classes.metaMaskIconContainer,
+    },
+    {
+      name: "Walletconnect",
+      description: "Connect using mobile wallet",
+      Icon: WalletConnectIcon,
+      iconContainerClassName: classes.walletConnectIconContainer,
+    },
+  ];
+
+  return (
+    <>
+      <PolygonHermezLogo className={classes.logo} />
+      <Typography type="body1" className={classes.appName}>
+        Bridge
+      </Typography>
+      <Card className={classes.card}>
+        <Typography type="h1" className={classes.cardHeader}>
+          Connect a wallet
+        </Typography>
+        <ul className={classes.walletList}>
+          {walletsData.map(({ name, description, Icon, iconContainerClassName }, i) => (
+            <li key={i} className={classes.wallet}>
```

Comment:
We should use different key than index to keep consistency when it may be not be used